.travis.yml

```diff
@@ -5,8 +5,11 @@ language: node_js
 matrix:
   include:
     - node_js: "4"
-      env: COVERALLS=1
     - node_js: "6"
+    - node_js: "7"
+      env: COVERALLS=1
 
 after_success:
-  - if [ "x$COVERALLS" = "x1" ]; then npm run coveralls; fi
+  - if [ "x$COVERALLS" = "x1" ] && npm i coveralls; then
```

Comment:
If you are insisting on freezing some particular version, then it's better to put coveralls into package.json